smack-core/src/main/java/org/jivesoftware/smack/ReconnectionManager.java

```diff
@@ -78,8 +79,8 @@ public void connectionCreated(XMPPConnection connection) {
      * 
      * @param enabled
      */
-    public static void setEnabledPerDefault(boolean enabled) {
-        enabledPerDefault = enabled;
+    public static void setEnabledByDefault(boolean enabled) {
```

Comment:
Please revert those changes, as they change existing public API (and are not related semantically to this commit).